cmd/vic-machine/common/limits.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"gopkg.in/urfave/cli.v1"
+
+	"github.com/vmware/govmomi/vim25/types"
+	"github.com/vmware/vic/pkg/flags"
+)
+
+type ResourceLimits struct {
+	VCHCPULimitsMHz       *int              `cmd:"cpu"`
```

Comment:
I discussed with @gigawhitlocks about the usage of this `*int`. If the user does not specify `--VCHCPULimitsMHz`, the value of this variable will be `nil`, so that we know that the user does not want to change/reconfigure the current value for this option. 

Alternatively, we can define `-1` as the default value of this option. So if the user doesn't specify a value for this option, its value will be `-1` instead of `nil`. The benefit is that we don't need an additional datatype `OptionalInt`.

What do you think?